index.js

```diff
@@ -1,36 +1,16 @@
 'use strict';
 
-const fastbootTransform = require('fastboot-transform');
-
 module.exports = {
   name: require('./package').name,
   included(app) {
     if (app.options && app.options.shepherd && app.options.shepherd.theme) {
-      this.theme = `dist/css/shepherd-theme-${app.options.shepherd.theme}.css`;
+      app.import(`node_modules/shepherd.js/dist/css/shepherd-theme-${app.options.shepherd.theme}.css`);
     }
-
     this._super.included.apply(this, arguments);
   },
   options: {
-    nodeAssets: {
```

Comment:
Now that this is removed, we should remove ember-cli-node-assets from package.json